shared/profile/user/weboftrust/index.tsx

```diff
@@ -31,6 +31,9 @@ const entry = (props: Props) => (
         />
       </Kb.Box2>
       <Kb.Box2 direction="vertical" fullWidth={true} style={styles.textContainer} centerChildren={false}>
+        <Kb.Text style={styles.attestationText} type="BodySmall">
+          {statusStatement}
+        </Kb.Text>
```

Comment:
Should talk about this

Reply:
lol. yes. this isn't great. totally agree. it's not to the design spec at all, i just found it very useful for testing and being sure that the correct attributes were flowing through for the correct reasons. i fully expect this to be cleaned up (both the code and the UI) before anyone sees it.